### ISSUE TITLE
Quest Broken - Investigate the Alchemist Shop

### ISSUE THREAD

**Issue by mantiblades** [SunstriderEmu/cc-buglist#13]
Investigate the Alchemist Shop Quest is broken.  Please review provided photo.
![QUEST BROKEN](https://user-images.githubusercontent.com/52470199/94718987-60494100-0307-11eb-96d5-8b50cb2fa2bb.png)

**Comment by mantiblades** [comment]
So I'm 27 lvl mage you need to be 26 lvl to do quest. I investigated the shop. Picked up both quest items. Went to my current pictured and provided location clicked the provided Manifestation scroll. creatures never manifested so I can not fill coffers to complete quest. mage Quest is broken. Tried logging out and back into game. Tried restarting the quest line. Need help

**Comment by kelno** [comment]
Hello, thanks for reporting. This is an old tracker used for something else. This is the actual bugtracker we use for Endless:
https://github.com/SunstriderEmu/BugTracker/issues

**Comment by kelno** [comment]
Oh I see you did already, great!

**Comment by mantiblades** [comment]
> Oh I see you did already, great!
I reposted the bug in the right place but it got closed without results. 
I'm not a tech I'm just trying to report a broken quest line so I can continue mage questline. 
I was told to come here to report it. 
The mage Quest: **Investigate the Alchemist Shop** when casting the manifestation scroll.. 
the scroll does not work... The creatures never show up. So I can't finish the quest. 
Do I need to complete this quest to go further into the mage questline?
Is there another server I can go to where the mage quests work.
or a GM in game that can help with that? 
I just want to play 
![QUEST BROKEN](https://user-images.githubusercontent.com/52470199/94850226-7cb6ad80-03db-11eb-8230-a09415a50862.png)
